### PR TITLE
chinese translate update

### PR DESCRIPTION
**Pull request by Ooopz**
Replace all Yaml to YAML, translate new added contents

### CODE REVIEW

File: src/lang/locale/zh-cn.ts [hunk 1]
```diff
@@ -740,44 +740,44 @@ export default {
...
-    'DEBUG': 'debug',
-    'INFO': '信息',
-    'WARN': '警告',
-    'SILENT': '静默',
+    'ERROR': 'ERROR',
+    'TRACE': 'TRACE',
+    'DEBUG': 'DEBUG',
+    'INFO': 'INFO',
+    'WARN': 'WARN',
+    'SILENT': 'SILENT',
```
Comment (pjkaufman) [inline src/lang/locale/zh-cn.ts:753]: Why did these change to their English values instead of being their Chinese equivalents?

Comment (Ooopz) [reply src/lang/locale/zh-cn.ts:753]: Just no need, programmers prefer to use the original English of programming terms, and I think debug level is only for programmers.

Comment (pjkaufman) [reply src/lang/locale/zh-cn.ts:753]: These options are not just meant for developers as it is something that is supposed to be used when reporting issues as well.

Comment (pjkaufman) [reply src/lang/locale/zh-cn.ts:753]: If that does not change anything, I am fine with merging this.

Comment (Ooopz) [reply src/lang/locale/zh-cn.ts:753]: If there is any explanation of different debug level, I think translate these to Chinese will be meaningful, but we just get some words here, so keep it not translated is fine.

Comment (pjkaufman) [reply src/lang/locale/zh-cn.ts:753]: Gocha. I will go ahead and merge this then.

**Pull request merged**